src/NativeScript/TNSRuntime+Inspector.mm

```diff
@@ -29,6 +29,8 @@
         : _messageHandler(Block_copy(handler)) {
     }
 
+    virtual ConnectionType connectionType() const override { return ConnectionType::Local; };
```

Comment:
I am not quite sure what is the semantic of the newly introduced `ConnectionType` enum and what value should be returned here (`Local` or `Remote`).

ping @fealebenpae @KristinaKoeva ?